src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
@whipermr5 do we need the `Test` prefix? We did not add such a prefix for the `teammates.test.driver.FieldValidator` class. It could be a bit confusing because we use use `Test` as a suffix for test classes. I'm ok to use the prefix but in that case we should add it for all such classes? What do you think?